docs_src/manuals/getting_started/code/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.12.4)
 
 project(GridTools-laplacian LANGUAGES CXX)
 
-find_package(GridTools 0.22.0 REQUIRED)
+find_package(GridTools 1.1.0 REQUIRED)
```

Comment:
Does anything prevent us from loading the version from version.txt here? Otherwise this would make sense I guess…